bridgesample/src/androidTest/java/com/livefront/bridgesample/scenario/util/CurrentActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.livefront.bridgesample.scenario.util
+
+import android.app.Activity
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.runner.lifecycle.ActivityLifecycleMonitorRegistry
+import androidx.test.runner.lifecycle.Stage.RESUMED
+
+/**
+ * @return the current activity being displayed
+ */
+fun getCurrentActivity(): Activity? {
```

Comment:
Is there ever going to be a case where this is actually null?

Reply:
No, it shouldn't be null or we might as well throw an exception now if the current activity doesn't exist. I improved the logic and added a couple of comments.